.codeclimate.yml

```diff
@@ -0,0 +1,12 @@
+version: '2'
+plugins:
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - javascript
+    exclude_patterns:
+      - '**/*.test.js'
```

Comment:
Should we also ignore `*.test.jsx` and `*.stories.jsx`?

Reply:
@mykeels done